src/assets/CollectibleDetectionController.ts

```diff
@@ -312,72 +316,71 @@ export class CollectibleDetectionController extends BaseController<
       return;
     }
 
-    await safelyExecute(async () => {
```

Comment:
Not sure if we're prepared to actually remove this and have truly unexpected errors percolate beyond this? @Gudahtt

Reply:
Nothing ever follows this call in this repository, so it's perfectly safe here. It is called in the products, so this change is probably worth calling out in the changelog. In the extension for example, it looks like there is a call in `actions.js` that could result in the loading screen getting stuck if this throws. That was already fallible though because of the connection... so that was always a risk, but now more so with this change.

I do think removing this is a good idea, if there are no cases you can anticipate where this would throw in production.